apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js

```diff
@@ -436,6 +442,48 @@ describe('DetailViewContents', () => {
       });
     }
 
+    it('appends auto email text if set to true', () => {
```

Comment:
how about a bit more descriptive name 'appends auto email text if emails are sent by the system'